server/lib/payments.js

```diff
@@ -578,12 +578,14 @@ const sendOrderConfirmedEmail = async (order, transaction) => {
       }
     }
 
-    const emailOptions = {
-      from: `${collective.name} <no-reply@${collective.slug}.opencollective.com>`,
-      attachments,
-    };
+    if (fromCollective.ParentCollectiveId !== collective.id) {
```

Comment:
You should add a comment to explain why we're doing this.

Reply:
@znarf : I've added a comment explaining the logic. Thanks. 👍🏽